controllers/chat/group/leaveGroupMember.js

```diff
@@ -1,12 +1,19 @@
+const {StreamChat} = require('stream-chat');
+const Environment = require('../../../config/environment');
 const BetterSocialCore = require('../../../services/bettersocial');
 const {ResponseSuccess} = require('../../../utils/Responses');
 const ErrorResponse = require('../../../utils/response/ErrorResponse');
 const Getstream = require('../../../vendor/getstream');
+const UsersFunction = require('../../../databases/functions/users');
+const {User} = require('../../../databases/models');
+const {CHANNEL_TYPE_STRING} = require('../../../helpers/constants');
 
 const leaveGroupMember = async (req, res) => {
   const {userId} = req;
   const {channelId} = req.body;
 
+  const ownUser = await UsersFunction.findUserById(User, userId);
+
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new code introduces a function `UsersFunction.findUserById(User, userId)` to find the user by their ID. However, there is no error handling for the case where the user might not be found in the database. This could lead to unexpected behavior or errors later in the code if `ownUser` is undefined.

```diff
- const ownUser = await UsersFunction.findUserById(User, userId);
+ const ownUser = await UsersFunction.findUserById(User, userId);
+ if (!ownUser) {
+   return res.status(404).json({ message: 'User not found' });
+ }
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->